cinema/views.py

```diff
@@ -20,25 +22,47 @@
     OrderListSerializer,
 )
 
+from user.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
Ensure that the `IsAdminOrIfAuthenticatedReadOnly` permission class is correctly implemented and imported. This custom permission should allow read-only access to authenticated users and full access to admin users. Verify its logic in the `user/permissions.py` file to ensure it meets these requirements.